### ISSUE TITLE
Deep link parameter in Dynamic link lost

### ISSUE THREAD

**Issue by Sasha2018** [firebase/quickstart-unity#924]
* Unity editor version: _2019.3.3f1____
  * Firebase Unity SDK version: _7.0.2____
  * Source you installed the SDK: ___.unitypackage__ 
  * Problematic Firebase Component: __Dynamic Links__
  * Other Firebase Components in use: ___Storage__ Auth
  * Additional SDKs you are using: __None___ 
  * Platform you are using the Unity editor on: __Mac___ 
  * Platform you are targeting: __Android__
  * Scripting Runtime: __IL2CPP_and Mono__ 

I have make a simple implementation of dynamic links realization. Currently all works fine on real device, but on DynamicLinkReceiver I have accepted url with deep link configurated on FirebaseConsole, but no deep link that I putted as base link when use new DynamicLinkComponents(...).

So, in unity I configured a long dynamic link at code in that format: https://DOMAIN.page.link/LINK_NAME?link=URL_FORM_UNITY
 and on FirebaseConsole I have short url  DOMAIN.page.link with Deep link URL: URL_FROM_FIREBASE_CONSOLE
 After link clicked on Android device I handler event from  DynamicLinks.DynamicLinkReceived on that method:
  private void DynamicLinkReceiver(object sender, EventArgs args)
        {
            var url = (args as ReceivedDynamicLinkEventArgs).ReceivedDynamicLink.Url;
            //here on url I have information only about URL_FROM_FIREBASE_CONSOLE, but want have about URL_FORM_UNITY
            
   I have see a lot of information and currently it seems to be a bug, because here
  https://www.spaceotechnologies.com/create-firebase-dynamic-links-android-tutorial/
  https://www.xspdf.com/resolution/50499939.html          
 
          
It seems same problem as was described here https://github.com/firebase/quickstart-unity/issues/730 on last comments.

Have no error on project

When try to make long link with debug (by adding "&d=1") I have 3 warnings:
       -  no log option set for custom domain
       -   the custom parameter (link) is a reserved parameter
       -   there is no configuration to prevent phishing on this domain. ***)

#### Steps to reproduce:

100%

#### Relevant Code:

Basically, you can check it on your project https://github.com/firebase/quickstart-unity/tree/master/dynamic_links/testapp, just make configuration to FirebaseConsole with changing domain and try to receive link on device

**Comment by Sasha2018** [comment]
Have solution! In case, if my experience will useful for someone:

1. It was misunderstanding - you can not use configured on console firebase dynamic link and change deep link on code. Does not matter, how you configured on code your dynamic link, if your dynamic link contains url of created on console dynamic link. You need use only URL prefix for customisation
2. Another problem was on adding to android parameters min version. After removing it, custom dynamic links working good. Maybe, it`s because my app still not on play store. I don`t really now why, but that work for me
<img width="758" alt="Screenshot 2021-01-19 at 00 59 21" src="https://user-images.githubusercontent.com/37545340/104969624-b1359000-59f1-11eb-8072-615b55659ada.png">
